pkg/apis/mysql/v1alpha1/helpers.go

```diff
@@ -50,10 +52,10 @@ func getOperatorVersionLabel(labelMap map[string]string) string {
 	return labelMap[constants.MySQLOperatorVersionLabel]
 }
 
-// EnsureDefaults will ensure that if a user omits and fields in the
+// EnsureDefaults will ensure that if a user omits any fields in the
 // spec that are required, we set some sensible defaults.
-// For example a user can choose to omit the version
-// and number of members.
+// For example a user can choose to omit the version and number of
+// members. MySQL server image default is defined through operator spec.
 func (c *Cluster) EnsureDefaults() *Cluster {
```

Comment:
We should probably default `.spec.repository` if it's not set (so that its value stored in the CRD), however, we'll need to default it based on the value of the flag which might prove problematic.